cfy_manager/components/restservice/config/haproxy.cfg

```diff
@@ -0,0 +1,23 @@
+global
+    maxconn 100
+defaults
+    log global
+    mode tcp
+    retries 2
+    timeout client 30m
+    timeout connect 4s
+    timeout server 30m
+    timeout check 5s
+listen stats
+    mode http
+    bind 127.0.0.1:7000
+    stats enable
+    stats uri /
+listen postgres
+    bind 127.0.0.1:5432
+    option httpchk
+    http-check expect status 200
+    default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
+{% for node in postgresql_server.cluster.nodes %}
+    server postgresql_{{ node }}_5432 {{ node }}:5432 maxconn 100 check check-ssl port 8008 ca-file /etc/haproxy/ca.crt
```

Comment:
there's no auth for that patroni status endpoint?

Reply:
Not for the status endpoint, but the information it provides is reasonably limited.
However, we should note in the documentation that it should be firewalled off anyway, to avoid people DoSing it if nothing else.